Conceptpower+Spring/src/main/java/edu/asu/conceptpower/root/DatabaseClient.java

```diff
@@ -309,6 +377,20 @@ public boolean match(ConceptList dict) {
         return allResults;
     }
 
+    @Override
+    public List<ConceptEntry> getAllEntriesByTypeId(String typeId) {
+        ConceptEntry entry = new ConceptEntry();
+        entry.setTypeId(typeId);
+
+        List<ConceptEntry> results = wordnetCacheClient.queryByExample(entry);
```

Comment:
looking at the class, it seems like the wordnetcache is not used (I originally used it to improve the search, but now we have lucene, so this is not needed anymore). The code that would store concepts in the cache is commented out. This line would therefore never return any results. You should be able to remove any queries to the wordnet cache.